python-lib/onedrive_client.py

```diff
@@ -121,11 +129,41 @@ def rename(self, from_path, to_path):
         return True
 
     def get_item(self, path):
+        if self.drive_id:
+            return self.get_drive_item(path)
         headers = self.generate_header()
-        response = requests.get(self.get_path_endpoint(path), headers=headers)
+        endpoint = self.get_path_endpoint(path)
+        response = requests.get(endpoint, headers=headers)
         onedrive_item = OneDriveItem(response.json())
         return onedrive_item
 
+    def get_drive_item(self, path):
+        item_path, item_name = os.path.split(path.strip("/"))
```

Comment:
item_name is not used, it can be replaced with _

Reply:
Done in 9960cc2